distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestRegistry.java

```diff
@@ -130,6 +118,26 @@ public void testCswRegistryUpdate() throws Exception {
     }
 
     @Test
+    public void testCswRegistryUpdateFailure() throws Exception {
+        String id = createRegistryEntry("urn:uuid:2014ca7f59ac46f495e32b4a67a51280");
+
+        given().body(Library.getCswRegistryUpdate()
```

Comment:
We may want to rethink the way we're setting specific values in the response. Using `replace` and `replaceAll` makes the tests brittle since minor changes to the canned response could cause the test to break for no apparent reason. It also introduces a lot of code duplication.